src/test_l1_plat_power.c

```diff
@@ -690,7 +692,7 @@ void test_l1_plat_power_negative_PLAT_API_GetWakeupSrc (void)
 {
```

Comment:
Line 568: in test_l1_plat_power_negative_PLAT_API_SetWakeupSrc API use PWRMGR_WAKEUPSRC_IR instead of PWRMGR_WAKEUPSRC_VOICE

Reply:
Done

Reply:
please address all places in the api

Reply:
Done